### PR TITLE
Track panel settings design tweaks

### PR DESCRIPTION
**Pull request by jyothishnt**
## Type

- [ ] Bug fix
- [ ] New feature
- [x] Improvement to existing feature

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-163
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-164
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-165
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-166

## Importance
Minor

## Description
Applying design tweaks from Andrea.

## Views affected
Genome browser track panel settings
As foundation black is changed to use ens-black, you could see small difference in black texts everywhere in the site.

- [ ] I have checked any requirements for Google Analytics  N/A
- [ ] I have created any required tests  N/A

### CODE REVIEW

File: src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx [hunk 1]
```diff
@@ -98,28 +99,41 @@ const BrowserTrackConfig: FunctionComponent<BrowserTrackConfigProps> = (
     props.updateApplyToAll(!applyToAll);
   }, [applyToAll, updateApplyToAll]);
 
+  const checkboxStyles: any = {
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx]: not `any`, please. Any is the last resort (and even then perhaps `unknown` is better, I don't know). Here the type is `{[key: string]: string}`. But it doesn't look like you even need to declare any type here; typescript should be capable of inferring it by itself.

File: src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx [hunk 2]
```diff
@@ -98,28 +99,41 @@ const BrowserTrackConfig: FunctionComponent<BrowserTrackConfigProps> = (
...
             onChange={applyToAllToggle}
           />
-          <label htmlFor="">All tracks</label>
         </dd>
       </dl>
       <dl>
         <dd>
-          <label htmlFor="">Track name</label>
-          <button className={styles.trackConfigSlider} onClick={toggleName}>
+          <label htmlFor={'trackConfig-trackName'}>Track name</label>
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx]: The `htmlFor` attribute on a label (which translates into the `for` attribute of the [html `label` tag](https://www.w3schools.com/tags/tag_label.asp) makes sense only if we are binding the label to an element with the corresponding id (usually — to an input). Here we are not; so it's useless.

Frankly, I do not understand this makrup. Why did Imran use the definition list (dl, dd) tags, when it's not a definition list? Why are we using label tags, although there is no corresponding input or select? I would have changed it to just divs or spans. Not necessarily now; perhaps during a later refactor.

Comment (jyothishnt) [reply src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx]: Regarding label `for`: In our case I added it to turn on/off trackConfigSlider when clicking on label itself, just like any other input. 

dd/dl:  True that it is not a def list. Divs are better in this case. I dont think we need to wait for refactoring. Lets clean it as we move.

Comment (jyothishnt) [reply src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx]: Changes pushed.

File: src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx [hunk 3]
```diff
@@ -98,28 +99,40 @@ const BrowserTrackConfig: FunctionComponent<BrowserTrackConfigProps> = (
...
+          <button
+            id={'trackConfig-trackName'}
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx]: use simple string

**Pull request merged**